src/components/DisplayObject.js

```diff
@@ -0,0 +1,133 @@
+import invariant from 'fbjs/lib/invariant'
+import { DisplayObject } from 'pixi.js'
+import { PROPS_DISPLAY_OBJECT, PROPS_RESERVED } from '../utils/props'
+import { isPointType, parsePoint } from '../utils/pixi'
+import { not, hasKey } from '../helpers'
+
+DisplayObject.getReactEventHandlers = function() {
+  return DisplayObject._reactEventHandlers
+}
+
+DisplayObject.setReactEventHandlers = function(eventHandlers) {
```

Comment:
Lets use getters and setters, example:

```js
Object.defineProperty(DisplayObject, 'reactEventHandlers', {
  get: function () {
    return DisplayObject._reactEventHandlers || [];
  },
  set: function(handlers) {
    Object.assign(DisplayObject, {
      _reactEventHandlers: handlers,
      _reactFilterProps: not(hasKey([...Object.keys(PROPS_RESERVED), ...handlers]))
    })
  }
})
```

Usage:

```js
DisplayObject.reactEventHandlers = [];
// or 
DisplayObject.reactEventHandlers.push('custom-event');

// maybe have a reset? 
DisplayObject.resetReactEventHandlers();
```

Reply:
I can fix that...